catt/controllers.py

```diff
@@ -221,7 +240,7 @@ def __init__(self, state_path, mode):
         elif mode == StateMode.ARBI:
             self._write_store({})
 
-    def get_data(self, name):
+    def get_data(self, name: str) -> str:
```

Comment:
I didn't intend for this to be included in the commit.

Reply:
Which do you mean?

Reply:
Just the change to this line.